proposals/0099-new-remote-control-modules-and-parameters.md

```diff
@@ -320,31 +320,57 @@ New LIGHT data types.
     <element name="REAR_LEFT_TURN_LIGHT" value="18"/>
     <element name="REAR_RIGHT_TURN_LIGHT" value="19"/>
     <element name="REAR_REGISTRATION_PLATE_LIGHT" value="20"/>
-    
+
     <!-- Exterior Lights by common function groups 501~800 -->
-    <element name="HIGH_BEAMS" value="501"/>
-    <element name="LOW_BEAMS" value="502"/>
-    <element name="FOG_LIGHTS" value="503"/>
-    <element name="RUNNING_LIGHTS" value="504"/>
-    <element name="PARKING_LIGHTS" value="505"/>
-    <element name="BRAKE_LIGHTS" value="506"/>
+    <element name="HIGH_BEAMS" value="501">
+      <description>Include all high beam lights: front_left and front_right.</description>
+    </element>
+    <element name="LOW_BEAMS" value="502">
+      <description>Include all low beam lights: front_left and front_right.</description>
+    </element>
+    <element name="FOG_LIGHTS" value="503">
+      <description>Include all fog lights: front_left, front_right, rear_left and rear_right.</description>
+    </element>
+    <element name="RUNNING_LIGHTS" value="504">
+      <description>Include all daytime running lights: front, left and right.</description>
```

Comment:
@yang1070 should this be "front_left and front_right" to match the other descriptions?

Reply:
@theresalech  I updated the description to front_left and front_right. thank you.